omero/developers/json-api.rst

```diff
@@ -46,7 +46,7 @@ version number can be found in the header:
 
 ::
 
-    X-OMERO-ApiVersion: 0.1
+    Current Version : 0.2
```

Comment:
Has the HTTP header key been modified?

Reply:
Oooops - sorry, I was misreading and missed the "header" bit. Couldn't remember what the `X-OMERO-ApiVersion` referred to. Fixed now.